mkdocs/docs/experiments/categories/azure/azure-instance-stop.md

```diff
@@ -195,6 +200,44 @@ spec:
         # name of the resource group
         - name: RESOURCE_GROUP
           value: '<resource group of AZURE_INSTANCE_NAME>'
+        # accepts enable/disable value. default is disable
+        - name: SCALE_SET
+          value: 'disable'
+        - name: TOTAL_CHAOS_DURATION
+          VALUE: '60'
+```
+
+### Scale Set Instances
+
+It contains comma separated list of instance names subjected to instance stop chaos belonging to Scale Set or AKS. It can be tuned via `SCALE_SET` ENV.
+
+Use the following example to tune this:
+
+[embedmd]:# (https://raw.githubusercontent.com/litmuschaos/litmus/master/mkdocs/docs/experiments/categories/azure/azure-instance-stop/azure-instance.yaml yaml)
```

Comment:
Can we embed this file instead: https://github.com/litmuschaos/litmus/blob/a0ac61ed1507fea458e8d73e9bee04578a86f06e/mkdocs/docs/experiments/categories/azure/azure-instance-stop/azure-scale-set-instance.yaml?